delivery_ups_oca/tests/test_delivery_ups.py

```diff
@@ -81,6 +81,11 @@ def _create_sale_order(self):
         return sale
 
     def test_order_ups_rate_shipment(self):
+        usd = self.env.ref("base.USD")
+        eur = self.env.ref("base.EUR")
+        currency = self.env.ref("base.main_company").currency_id
+        currency_extra = eur if currency == usd else usd
+        self.sale.currency_id = currency_extra
```

Comment:
I think it would be better to create another test for this specific case instead of re-using this existing one. That way, it will be easier to avoid regression changes...